toolkit/scripts/srpm_pack.mk

```diff
@@ -50,30 +50,6 @@ $(BUILD_SRPMS_DIR): $(STATUS_FLAGS_DIR)/build_srpms.flag
 	@touch $@
 	@echo Finished updating $@
 
-ifeq ($(DOWNLOAD_SRPMS),y)
-$(STATUS_FLAGS_DIR)/build_srpms.flag: $(LOCAL_SPECS) $(LOCAL_SPEC_DIRS) $(SPECS_DIR)
```

Comment:
Is the idea here to remove the `DOWNLOAD_SRPMS` flag only until the [new srpmdownloader tool](https://github.com/rlmenge/CBL-Mariner/pull/23) is added? I'm asking because in general removing published public APIs/interfaces/features within one release of any project is usually discouraged. I doubt anyone except for us is using this flag, so it's probably not a big deal here, but to be on the safe side, I'd prefer to leave the flag in place and instead consider adding a warning, that this feature might be malfunctioning, but a fix is on the way. After all this still has a chance to work with derivative specs and `SRPM_URL_LIST` values. A minor benefit of doing it this way, is that we minimize chances of missing some documentation or functionality when we try to restore this feature later on.

I have nothing against removing `ALLOW_SRPM_DOWNLOAD_FAIL`. I think it doesn't do anything anyway.